manifests/init.pp

```diff
@@ -1,20 +1,49 @@
+# == Class: homebrew
+#
+# This class is able to install and configure homebrew and brew cask on node.
+#
+# === Parameters
+#
+# [*user*]
+#   User owner of the brew files and folders.
+#   Defaults to nothing.
```

Comment:
If user is set but doesn't exist on the system it will also fail. Should we create the user if doesn't exist but is specified on the config?